yeonjae/programmers-1/두정수사이의합.js

```diff
@@ -0,0 +1,19 @@
+function solution(a, b) {
+  let answer = 0;
+  let max = Math.max(a, b);
+  let min = Math.min(a, b);
+
+  for (let i = min; i <= max; i++) {
+    answer += i;
+  }
+  return answer;
+}
```

Comment:
```suggestion
function solution(a, b) {
  const maxNum = Math.max(a, b);
  const minNum = Math.min(a, b);
  let answer = 0;

  for (let i = min; i <= max; i++) {
    answer += i;
  }
  return answer;
}
```
- 고생하셨습니다 👍 변수 선언 시 max, min은 값이 바뀌지 않는 상수기때문에 const 선언과 max, min보다는 suffix로 Num을 붙여주는게 더 명확할 것 같다는 생각이 들었습니다 🙆‍♂️